src/TestFramework/AbstractTestFrameworkAdapter.php

```diff
@@ -229,17 +227,20 @@ public function getInitialTestsFailRecommendations(string $commandLine): string
      *
      * @return string[]
      */
-    private function findPhp(bool $includeArgs = true): array
+    private function findPhp(): array
     {
-        if ($this->cachedPhpPath === null || $this->cachedIncludedArgs !== $includeArgs) {
-            $this->cachedIncludedArgs = $includeArgs;
-            $phpPath = (new PhpExecutableFinder())->find($includeArgs);
+        if ($this->cachedPhpPath === null) {
```

Comment:
it's not a php path tbh. Probably we should rename this property.

also we should drop `$cachedIncludedArgs` as it's no longer required